### PR TITLE
Fix false positives for prefers-contrast in media-feature-name-no-unknown

### LINKED ISSUE #5427: Fix false positives for prefers-contrast in media-feature-name-no-unknown
> What is the problem you're trying to solve?

The `media-feature-name-no-unknown` rule doesn't allow the new [`prefers-contrast`](https://drafts.csswg.org/mediaqueries-5/#prefers-contrast) media query feature. This is a new media query added to CSS Media Queries Level 5 to utilize the contrast preference operating systems provide. This new media query is implemented in [Safari Technology Preview](https://bugs.webkit.org/show_bug.cgi?id=215507) and [Firefox Nightly](https://hacks.mozilla.org/2020/07/adding-prefers-contrast-to-firefox/) and is currently in development for [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=1107431).

> What solution would you like to see?

The media query `prefers-contrast` to be allowed by `media-feature-name-no-unknown` rule.

### PR DESCRIPTION
**Pull request by danrichman**
This is a new media query added to CSS Media Queries Level 5 to utilize the contrast preference operating systems provide.

> Which issue, if any, is this issue related to?

Closes #5427

> Is there anything in the PR that needs further explanation?

Please see issue #5427

### CODE REVIEW

File: lib/reference/keywordSets.js [hunk 1]
```diff
@@ -608,6 +608,7 @@ keywordSets.mediaFeatureNames = uniteSets(keywordSets.deprecatedMediaFeatureName
 	'overflow-inline',
 	'pointer',
 	'prefers-color-scheme',
+	'prefers-contrast'
```
Comment (jeddy3) [inline lib/reference/keywordSets.js:611]:
```suggestion
	'prefers-contrast',
```

**Review by jeddy3 — CHANGES_REQUESTED** [review]
Thanks for the pull request, and for using the templates here and in the issue.

I've requested a small change to fix a typo.

**Review by jeddy3 — APPROVED** [review]
Thanks, for committing the suggestion.

LGTM. 

This is being merged into [`v14` that we need help getting out the door](https://github.com/stylelint/stylelint/issues/5205). In the meantime, you can use disable comments or the [`ignoreMediaFeatureNames` secondary option](https://stylelint.io/user-guide/rules/list/media-feature-name-no-unknown/#ignoremediafeaturenames-regex-regex-string).

**Review by ybiquitous — APPROVED** [review]
Thanks. LGTM 👍🏼

**Comment by jeddy3** [comment]
Changelog entry added:

> - Fixed: `media-feature-name-no-unknown` false positives for `prefers-contrast` ([#5428](https://github.com/stylelint/stylelint/pull/5428)).

**Pull request merged**